pkg/kubelet/server/server.go

```diff
@@ -282,7 +284,7 @@ func (s *Server) InstallDefaultHandlers() {
 const pprofBasePath = "/debug/pprof/"
 
 // InstallDeguggingHandlers registers the HTTP request patterns that serve logs or run commands/containers
-func (s *Server) InstallDebuggingHandlers() {
+func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
```

Comment:
I disagree. This is very specific to the in-process docker runtime shim, and doesn't apply for any of the other runtimes. Adding it to the runtime interface would mean adding a no-op (return a nil handler) or respond with a 404 to all the other runtimes. We could add a getter to the host interface, but I'm indifferent about which is cleaner...